ministryofjustice/laa-apply-for-criminal-legal-aid
# CRIMAP-379 Implement prometheus exporter

## Description of change
First round on the prometheus exporter.

It is deployed on its own pod (single replica), exposing a `/metrics` endpoint which is then scrapped by the cloud-platform.

Counterpart CP PR here:
https://github.com/ministryofjustice/cloud-platform-environments/pull/14075

## Link to relevant ticket
https://dsdmoj.atlassian.net/browse/CRIMAP-414

## Notes for reviewer

## How to manually test the feature
It can be "tested" on localhost by adding `PROMETHEUS_EXPORTER_HOST=localhost` to your `.env.development.local` and running the server in one terminal and the prometheus exporter in another terminal.
Then you can go to `http://localhost:9394/metrics` to see the metrics, and navigate the service to produce new metrics that are fetched periodically.
